CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ project(Ship VERSION 8.0.6 LANGUAGES C CXX)
 include(CMake/soh-cvars.cmake)
 include(CMake/lus-cvars.cmake)
 
+option(SUPPRESS_WARNINGS "Suppress warnings in LUS and src (decomp)" ON)
+if(SUPPRESS_WARNINGS)
+    MESSAGE("Suppressing warnings in LUS and src")
+    if(MSVC)
+        set(WARNING_OVERRIDE /w STRING "")
```

Comment:
Setting `STRING ""` (or specifically `<type> <docstring>`) is only valid after the `CACHE` keyword. Since we don't want this to be a cache variable, you need to drop these, as they are being considered as part of the normal variable value now (hence the build issue).